packages/flutter/test/material/dropdown_test.dart

```diff
@@ -174,6 +176,41 @@ void verifyPaintedShadow(Finder customPaint, int elevation) {
   );
 }
 
+Future<void> checkDropdownColor(WidgetTester tester, {Color color}) async {
+  const String text = 'foo';
+  await tester.pumpWidget(
+    MaterialApp(
+      home: Material(
```

Comment:
Is Material required?

Reply:
MaterialApp can be replaced by TestApp and Material can be replaced by Scaffold, but it appears to be required.